app/src/main/java/com/sdp/movemeet/ui/workout/Model.java

```diff
@@ -0,0 +1,27 @@
+package com.sdp.movemeet.ui.workout;
+
+public class Model {
```

Comment:
This is the model for the workout ?

Reply:
Yes, exactly! It describes a Workout with a name, description and icon.